packages/stream-text/src/index.ts

```diff
@@ -82,9 +82,29 @@ export interface StreamTextResult {
 export interface StreamTextStep {
   choices: StreamTextChoice[]
   messages: Message[]
+  toolResult?: StreamTextToolResult
```

Comment:
Can we use `GenerateTextToolResult`? (maybe put it in `@xsai/shared-chat`)

Reply:
great point, let me fix it